src/utils/validateIdentifierInputs.ts

```diff
@@ -19,25 +19,21 @@ export const validateIdentifierInputs = (
   identifierInputs: IdentifierInput[]
 ): boolean => {
   for (const { value, validationRegex, textInputType } of identifierInputs) {
-    if (
-      textInputType === "PAYMENT_RECEIPT" &&
-      (!value ||
-        !isMatchRegex(
-          value,
-          validationRegex
-            ? validationRegex
-            : defaultPaymentReceiptValidationRegex
-        ))
-    ) {
-      throw new Error(ERROR_MESSAGE.INVALID_PAYMENT_RECEIPT_NUMBER);
-    }
-    if (!value) {
+    if (textInputType !== "PAYMENT_RECEIPT" && !value) {
```

Comment:
realised that altho the backend allows for optional fields for different text inputs types, this current logic only allows optional payment receipts. shall we address this in another story?

Reply:
sounds good! But the PR of that story would be the same as the PR676?

Reply:
or shall we address this inside PR676 haha

Reply:
actually another question, is there a reason why we're doing an additional check here of  whether `textInputType !== "PAYMENT_RECEIPT"` haha

Reply:
as discussed,, need the check cause we only allow `PAYMENT_RECEIPT` to be empty currently!

Reply:
ok!